src/styles/sidebar/components/annotation-license.scss

```diff
@@ -0,0 +1,23 @@
+@use "../../variables" as var;
+
+.annotation-license {
+  @include var.font-small;
+  border-top: 1px solid var.$grey-3;
+  padding: 0.5em 0;
```

Comment:
On master the white space above and below the dividing line is balanced:

<img width="417" alt="Screenshot 2019-12-04 07 33 17" src="https://user-images.githubusercontent.com/2458/70122360-68ddd600-1668-11ea-8431-31b53a45ee38.png">

On this branch it is uneven:

<img width="404" alt="Screenshot 2019-12-04 07 34 11" src="https://user-images.githubusercontent.com/2458/70122395-785d1f00-1668-11ea-89af-579f672ce379.png">

Is that intentional?

Reply:
Hum. That may be the overall effect, but this component itself has equal padding:

![image](https://user-images.githubusercontent.com/439947/70152398-240f6a80-167b-11ea-8ce1-b4445081b213.png)

![image](https://user-images.githubusercontent.com/439947/70152414-2d98d280-167b-11ea-9d25-7595f67810eb.png)

Before I adjusted it, the CSS had a fairly insane rule:

`padding-top: 0.583em;`

It is my sense that the overall outer rhythm should be in the province of the containing elements/component, in this case, `annotation` (i.e. adjustments should be made in that component's CSS, not the `AnnotationLicense` CSS). Does that sound right to you?

Reply:
e.g. I notice that the `annotation-footer` styling has a (rather inelegant) top margin of `7px` but no corresponding bottom margin:

![image](https://user-images.githubusercontent.com/439947/70152644-98e2a480-167b-11ea-80a0-d075b57e6c57.png)

Reply:
> It is my sense that the overall outer rhythm should be in the province of the containing elements/component, in this case, annotation (i.e. adjustments should be made in that component's CSS, not the AnnotationLicense CSS). Does that sound right to you?

Yes, I agree.

Reply:
OK, I've dropped too much time into this. The vertical rhythm of the various pieces of the annotation card is uneven, and it's outside of the scope of this change to address it. For now I've removed the bottom padding so that this text "appears" to be vertically centered as it was before.